src/lib/providers.ts

```diff
@@ -16,15 +17,20 @@ export const getAvailableProviders = async () => {
           openAIApiKey,
           modelName: 'gpt-3.5-turbo',
           temperature: 0.7,
```

Comment:
Can you add the baseURL here as well please

Reply:
thanks for you response . 

do you mean add  baseURL in  async method like this 

`export const getAvailableProviders = async ({baseuri}) => {`

i had  add  baseURL in ChatOpenAI configuration

```

models['openai'] = {
        'gpt-3.5-turbo': new ChatOpenAI({
          openAIApiKey,
          modelName: 'gpt-3.5-turbo',
          temperature: 0.7,
          configuration : {
            baseURL : openAIBaseUrl,
          }
        }),
```